__tests__/app.test.js

```diff
@@ -173,6 +206,50 @@ describe('appTests', () => {
                 })
         })
             // /api/reviews/review_id/comments 
-        
+        test('POST: 404 /api/reviews/:review_id/comments Rejects a well formed post to an existing review however for a non-existant user.', () => {
+
+            const testComment = {
+                "username": "I do not exist",
+                "body": "this game really tested me"
+            }
+
+            return request(app)
+                .post('/api/reviews/1/comments')
+                .send(testComment)
+                .expect(404)
+                .then(({body}) => {
+                    expect(body.msg).toBe('No user found with this name: I do not exist')
+                })
+        })
+
+        test('POST: 404 /api/reviews/:non_existant_review_id/comments Rejects a valid post to a well formed but non existant route', () => {
+            const testComment = {
+                "username": 'bainesface',
+                "body": 'I think this test game is amazing'
+            }
+
+            return request(app)
+                .post('/api/reviews/99999999/comments')
+                .send(testComment)
+                .expect(404)
+                .then(({body}) => {
+                    expect(body.msg).toBe('No review found for this ID: 99999999')
+                })
+        })
+
+        test('POST: 400 Rejects a valid post, to a badly formed route', () => {
+            const testComment = {
+                "username": 'bainesface',
+                "body": 'I think this test game is amazing'
+            }
+
+            return request(app)
+                .post('/api/reviews/impossible_route/comments')
+                .send(testComment)
+                .expect(400)
+                .then(({body}) => {
+                    expect(body.msg).toBe('Invalid Input: bad review ID')
+                })
+        })
```

Comment:
Could we just add a couple more tests here please? One for a 201 that ignores unnecessary properties, and one for a 400 when there's missing required properties (i.e. no username or body)